src/config/config.c

```diff
@@ -65,11 +65,27 @@ static int read_main_elements(const OS_XML *xml, int modules,
 
         chld_node = OS_GetElementsbyNode(xml, node[i]);
 
-        if (chld_node && (strcmp(node[i]->element, osglobal) == 0)) {
-            if (((modules & CGLOBAL) || (modules & CMAIL))
-                    && (Read_Global(chld_node, d1, d2) < 0)) {
+        /* Common configuration */
+		if (chld_node && (strcmp(node[i]->element, oslabels) == 0)) {
```

Comment:
Remove extra tabs present in all this file.

Reply:
Done